src/ImageSharp/Image/ImageBase{TPixel}.cs

```diff
@@ -37,6 +38,11 @@ public abstract class ImageBase<TPixel> : IImageBase<TPixel>
         private TPixel[] pixelBuffer;
 
         /// <summary>
+        /// The span representing the pixel buffer
+        /// </summary>
+        private Span<TPixel> span;
```

Comment:
Ah yes, of course!

I'm happy to change pixels to `Span<TPixel>` makes sense and certainly does no harm.